interfaces/field.ts

```diff
@@ -0,0 +1,20 @@
+export default interface Field {
+  field_class?: string;
+  attributes?: {
+    rows?: Number | null;
+  };
+  label?: string;
+  name: string;
+  title?: string;
+  fm?: {
+    add_more_label: string;
```

Comment:
Here and above, in our ESLint standards, we should be using camel case instead of snake case, unless you're getting these properties from somewhere that you don't have control over (like an API). (Going a bit further, even if this comes from a custom API endpoint that you do have control over, it's preferred to use snake case for API responses, so this would be fine.)